syncer/find_fork_test.go

```diff
@@ -74,48 +74,41 @@ func TestForkFinder_Purge(t *testing.T) {
 	require.Equal(t, 0, tf.NumPeersCached())
 }
 
-func createPeerHashes(max uint32) []types.Hash32 {
-	peerHashes := make([]types.Hash32, max+1)
-	gLid := types.GetEffectiveGenesis()
-	for i := uint32(0); i <= max; i++ {
-		lid := types.NewLayerID(i)
-		if lid.Before(gLid) {
-			peerHashes[i] = types.Hash32{}
-		} else {
-			peerHashes[i] = types.RandomHash()
-		}
+func layerHash(layer int, good bool) types.Hash32 {
+	if good {
+		return types.BytesToHash([]byte(strconv.Itoa(layer)))
 	}
-	return peerHashes
+	return types.BytesToHash([]byte(strconv.Itoa(2 * layer)))
```

Comment:
do you want to use even as bad, and odd as good? is it ok that good=2, will be equal to bad=1?

maybe prefixing them will be more obvious 
  
```
var h2 types.Hash32
h := h2[:0]
h = append(h, "good/"...)
binary.LittleEndian.AppendUint32(h, 72)
```

Reply:
thanks for sharing. done.

good here implies that the peer always has the correct layer hash, and local node has the incorrect one.